src/Analyzers/CSharp/Analyzers/UseCollectionExpression/UseCollectionExpressionHelpers.cs

```diff
@@ -733,11 +733,20 @@ or ImplicitStackAllocArrayCreationExpressionSyntax
                 .WithPrependedLeadingTrivia(ElasticMarker);
     }
 
+    private static bool HasStructuredTriviaBetweenCloseTokenAndInitializer(InitializerExpressionSyntax initializer)
```

Comment:
note: this name is not correct.   'structured trivia' means something specific in Roslyn.  Specifically it is things like doc-comments or pp-directives.  Trivia that have parse trees under them themselves.  This function is asking "is there single or multiline trivia before this node"